SQL_Analysis_Ninuca_Jintcharadze_WindFrames_HW.sql

```diff
@@ -0,0 +1,115 @@
+--1.annual sales analysis by channel and region.
+WITH TotalSales AS (
+    SELECT co.country_region, t.calendar_year, ch.channel_desc, SUM(s.amount_sold) AS amount_sold
+    FROM sh.sales s
+    JOIN sh.customers c ON s.cust_id = c.cust_id
+    JOIN sh.times t ON s.time_id = t.time_id
+    JOIN sh.countries co ON c.country_id = co.country_id
+    JOIN sh.channels ch ON s.channel_id = ch.channel_id  
+    WHERE t.calendar_year BETWEEN 1999 AND 2001 AND co.country_region IN ('Americas', 'Asia', 'Europe')
```

Comment:
In order to calculate the previous period, you have to select data for 1998 year, then filter necessary rows